.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+version: 2.1
+
+defaults: &defaults
```

Comment:
Maybe you could use circleci built-in [executors](https://circleci.com/docs/configuration-reference#executors-requires-version-21) to define container for writeing it to make it readable than `default` and YAML anchors and alias syntax, also `executors` could provide multiple executor if we need in the furture.

```yaml
executors:
   node-executor:
       working_directory: ~/repo
       docker:
          - image: cimg/node:16.15.1

install:
    executor: node-executor
    steps:
      - checkout

lint:
    executor: node-executor
    steps:
      - checkout
```

Reply:
It looks great to use executors! I'll update them.